xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -752,6 +752,9 @@ core.footer.modification=Last modified by {0} on {1}
 ## This label is only visible by AT users to help with understanding the meaning of the top section containing
 ## the page title and the info about the last modification of the page.
 core.document.header.info.label=Page header
+## These two text alternatives are only visible by AT users.
+core.document.header.companylogo.anchor.text=Wiki home
+core.document.header.companylogo.logo.text=Logo of the wiki
```

Comment:
I'm wondering some things:

1. Should we really use the name `companylogo`? I see that it is the ID of the container, but in my opinion it isn't really fitting as the wiki doesn't necessarily belong to a company. What about a shorter "logo" and then "image" instead of the "logo" for the second key?
2. Should we really name the text "Wiki home", why don't we keep the shorter "Home" that was the title previously? Asides from being shorter, there is also the aspect that from the perspective of the user, this website might not really be a wiki but some public, non-editable website.
3. While the logo alternative text seems good enough, I wonder if we shouldn't offer an easy way to configure the alternative text next to the logo upload.

Reply:
1. Yeah, company is definitely off, `orglogo` could work too. Implemented your suggestion in https://github.com/xwiki/xwiki-platform/pull/3510/commits/34c40f83e0cd1f927503c6e7c1346a59adffe0b9 :+1: 
2. My reason was that in the context of a browser, the `Home` is something else more generic and it'd be better to have something a bit more precise than this. I forgot that `wiki` is not always what the end user sees. Reverting this change to go with your suggestion in https://github.com/xwiki/xwiki-platform/pull/3510/commits/defdf3df05f6389ffa714348770e684779eb0ca2 :+1: 
3. Created https://jira.xwiki.org/browse/XWIKI-22602 to track the progress on this idea :+1: For now, admins can still edit it with a translation, which is quite easy once they figure out the translation they need to edit :)